ui/src/app/groupings/[groupingPath]/_components/export-dropdown.tsx

```diff
@@ -1,16 +1,211 @@
 'use client';
 
-import { Button } from '@/components/ui/button';
-import { FileInput, ChevronDown } from 'lucide-react';
+import { useReducer, useRef } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faFileExport,
+    faCaretDown,
+    faUsers,
+    faUserPlus,
+    faUserMinus,
+    faIdCard
+} from '@fortawesome/free-solid-svg-icons';
+import { z } from 'zod';
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip';
+
+const memberSchema = z.object({
+    firstName: z.string(),
+    lastName: z.string(),
+    uid: z.string(),
+    uhUuid: z.string()
+});
+
+const groupingMemberSchema = z.object({
+    allMembers: z.object({
+        members: z.array(memberSchema)
+    }),
+    groupingInclude: z.object({
+        members: z.array(memberSchema)
+    }),
+    groupingExclude: z.object({
+        members: z.array(memberSchema)
+    }),
+    groupingBasis: z.object({
+        members: z.array(memberSchema)
+    })
+});
+
+type Member = z.infer<typeof memberSchema>;
+type GroupingMembers = z.infer<typeof groupingMemberSchema>;
```

Comment:
I noticed you are using zod but not using it for any data validation. Since that is the case, you can just simply define the types without zod.